### PR TITLE
Updated the chart to support using the new 0.3.1-PROXY image to use trow as a proxy for docker hub

### PR DESCRIPTION
**Pull request by 22ndtech-admin**
I updated the chart version and the appVersion. I also added values to the values.yaml and parameters to the statefulset.yaml  to accept them for passing the image tag and the --proxy-docker-hub arguments.

### CODE REVIEW

**Comment by amouat** [comment]
Sorry I missed this! This looks good - let me get is tested and get back to you.

File: charts/trow/values.yaml [hunk 1]
```diff
@@ -20,6 +21,7 @@ trow:
     allowImages: []
     disallowLocalPrefixes: []
     disallowLocalImages: []
+    proxyDockerHub: true
```
Comment (amouat) [inline charts/trow/values.yaml:24]: I think it should probably be false by default

**Review by amouat — CHANGES_REQUESTED** [review]

File: charts/trow/values.yaml (hunk 1 above)
Comment (22ndtech-admin) [reply charts/trow/values.yaml]: Ok, I changed it to false

**Review by amouat — APPROVED** [review]

**Comment by amouat** [comment]
Merged, apologies for the delay.

Many thanks for the PR!

**Pull request merged**